### PR TITLE
COST-803: Grab the start/end dates from the pod usage report for the manifest

### PR DESCRIPTION
**Pull request by abaiken**
As a result of this PR the manifest with the start/end dates looks like this: 
```
{
 "uuid": "1f4b287b-20cd-4d62-8bcd-df510b29b119",
 "cluster_id": "5997a261-f23e-45d1-8e01-ee3c765f3aec",
 "version": "0b364acf50be16863bca1e68508c8d90216b2566",
 "date": "2021-01-06T15:02:50.098166Z",
 "files": [
  "1f4b287b-20cd-4d62-8bcd-df510b29b119_openshift_usage_report.0.csv",
  "1f4b287b-20cd-4d62-8bcd-df510b29b119_openshift_usage_report.1.csv",
  "1f4b287b-20cd-4d62-8bcd-df510b29b119_openshift_usage_report.2.csv",
  "1f4b287b-20cd-4d62-8bcd-df510b29b119_openshift_usage_report.3.csv"
 ],
 "start": "2021-01-06T14:00:00Z",
 "end": "2021-01-06T14:59:59Z"
}
```

### CODE REVIEW

File: packaging/packaging.go [hunk 1]
```diff
@@ -364,7 +419,16 @@ func (p *FilePackager) PackageReports() error {
 	} else if err != nil {
 		return fmt.Errorf("PackageReports: %v", err)
 	}
-
+	// get the start and end dates from the report
+	log.Info("getting the start and end intervals for the manifest")
+	for _, file := range filesToPackage {
+		absPath := filepath.Join(p.DirCfg.Staging.Path, file.Name())
+		if strings.Contains(absPath, "pod") {
```
Comment (maskarb) [inline packaging/packaging.go:426]:
```suggestion
		if strings.Contains(file.Name(), "pod") {
			absPath := filepath.Join(p.DirCfg.Staging.Path, file.Name())
```

File: packaging/packaging.go [hunk 2]
```diff
@@ -49,6 +50,8 @@ type FilePackager struct {
 	uid      string
 	MaxSize  int64
 	maxBytes int64
+	Start    time.Time
+	End      time.Time
```
Comment (maskarb) [inline packaging/packaging.go:54]: These fields do not need to be exported, so let's lowercase them.

**Review by maskarb — COMMENTED** [review]
One comment and one NIT.

File: packaging/packaging.go (hunk 2 above)
Comment (abaiken) [reply packaging/packaging.go]: done!

**Review by maskarb — APPROVED** [review]

**Pull request merged**